examples/snippets/interleaved-tables-before-7.1/README.md

```diff
@@ -0,0 +1,167 @@
+# Sample - Interleaved Tables - Before Rails 7.1.0
```

Comment:
Are these changes (before 7.1) migrated from the previous code/doc?

Reply:
Yes, I just renamed the folder where they are in, but GitHub shows it like it's all new.